README.md

```diff
@@ -98,3 +98,75 @@ There are two options when using the Avalanche-CLI:
+- **Config Files**: Configure as you would with the regular AvalancheGo binary using config files. Mount the config directory with `-v ~/.avalanchego:/root/.avalanchego`.
+
+### Data Persistence
+
+To persist data across container restarts, you need to mount the `/root/.avalanchego` directory. For example, `-v ~/.avalanchego:/root/.avalanchego`. The container runs as root by default.
+
+### Updating
+
+Run `docker stop avago; docker rm avago;` then start a new container with the latest version tag in your `docker run` command.
+
+### Networking
+
+Using `--network host` is recommended to avoid any issues.
+If you know what you are doing, you will need port `AVAGO_STAKING_PORT` (default `9651`) open for the validator to connect to the subnet. For the RPC server, open `AVAGO_HTTP_PORT` (default `9650`). Do not attempt to remap `AVAGO_STAKING_PORT` using the Docker `-p` flag (e.g., `-p 9651:1234`); it will not work. Instead, set `AVAGO_STAKING_PORT=1234` and then use `-p 1234:1234`.
+
+This is because the staking port is used for peer-to-peer communication between validators, and the port number is part of the node's identity in the network. If you remap it using Docker's port mapping, other nodes will still try to connect to the original port number advertised by your node, leading to connection failures. The HTTP port can be remapped freely since it's only used for API calls.
+
+### Example Configs
+
+#### Fuji Subnet Validator
+
+```bash
+docker run -it -d \
+  --name avago \
+  --network host \
+  -v ~/.avalanchego:/home/avalanche/.avalanchego \
+  -e AVAGO_NETWORK_ID=fuji \
+  -e AVAGO_PARTIAL_SYNC_PRIMARY_NETWORK=true \
+  -e AVAGO_TRACK_SUBNETS=REPLACE_THIS_WITH_YOUR_SUBNET_ID \
+  -e AVAGO_PUBLIC_IP_RESOLUTION_SERVICE=ifconfigme \
+  avaplatform/subnet-evm:v0.7.1-rc.0
+```
+
+- `AVAGO_PARTIAL_SYNC_PRIMARY_NETWORK`: Ensures you don't sync the X and C-Chains.
+- `AVAGO_TRACK_SUBNETS`: Sets the subnet ID to track. It will track all chains in the subnet.
+- `AVAGO_NETWORK_ID=fuji`: Sets the network ID to Fuji. Remove to sync Mainnet.
+- `AVAGO_PUBLIC_IP_RESOLUTION_SERVICE=ifconfigme`: Required for AWS EC2 instances to be accessed from outside AWS.
+
+#### Fuji Subnet RPC
+
+```bash
+docker run -it -d \
+  --name rpc \
+  --network host \
+  -v ~/.avalanchego_rpc/:/home/avalanche/.avalanchego \
+  -e AVAGO_NETWORK_ID=fuji \
+  -e AVAGO_PARTIAL_SYNC_PRIMARY_NETWORK=true \
+  -e AVAGO_TRACK_SUBNETS=hk755meusfKqBb9C9RfzzCxZFkdSXhaFHTcbtycMUSQ11o2cd \
+  -e AVAGO_HTTP_PORT=8080 \
+  -e AVAGO_STAKING_PORT=9653 \
+  -e AVAGO_HTTP_ALLOWED_HOSTS="*" \
+  -e AVAGO_HTTP_HOST=0.0.0.0 \
+  -e AVAGO_PUBLIC_IP_RESOLUTION_SERVICE=ifconfigme \
+  avaplatform/subnet-evm:v0.7.1-rc.0
+```
+
+- `AVAGO_STAKING_PORT` is set to `9653` in case you want to run this on the same machine as the validator. Remove this to set to the default `9651`.
```

Comment:
I would leave `AVAGO_STAKING_PORT=9651` in the docker run command, and specify that the port can be changed in case of a conflict.